### ISSUE TITLE
Request to add ability to configure time for additional notifications

### ISSUE THREAD

**Issue by marco170475** [m-i-n-a-r/birday#350]
I would like the ability to configure different time for additional notifications; and an option to configure additional notification also for the event's day. An old birthday reminder app i used before discovering Birday, had options to set an additional notification also for the event's day. For my use case, this is useful because i like to have a reminder two days before the event at 07:00 PM, and, in the event's day, the first reminder at midnight and another one at 07:00 PM.

Anyway, thank you for working on this wonderful app!

**Comment by m-i-n-a-r** [comment]
That's quite tricky and it wasn't requested before, but thanks for suggesting it. If others request this feature, I'll try to implement it in one of the next updates 😄

**Comment by hariveerabahu** [comment]
yes this feature would be helpful, for sending the birthday wishes most of them have the habit of wishing them at midnight 12 AM. If there is a feature or option to set the time for additional notification it would be very helpful
For example: 
If birthday is on 26 September 2025. 
Main notification i will set it on the day of the birthday. 
Additional notification i will set it on 25 September 2025 11.00 PM.
